lib/graph3d/DataGroup.js

```diff
@@ -0,0 +1,240 @@
+var DataSet  = require('../DataSet');
+var DataView = require('../DataView');
+var Point3d  = require('./Point3d');
+var Range    = require('./Range');
+
+
+function DataGroup() {
```

Comment:
For all matters related to `jsdoc` annotations, I am assuming that http://usejsdoc.org is leading.